common/script/index.js

```diff
@@ -33,10 +33,6 @@ api.capByLevel = statHelpers.capByLevel;
 api.tnl = statHelpers.toNextLevel;
 api.diminishingReturns = statHelpers.diminishingReturns;
 
-// TODO under api.libs?
-import splitWhitespace from './libs/splitWhitespace';
-const $w = api.$w = splitWhitespace;
```

Comment:
No it wouldn't because the client files have are not being linted. To avoid the linter exception just do api.$w = splitWhitespace; without assign it to a local const